### PR TITLE
Complete week 9 assignment

### PR DESCRIPTION
**Pull request by Sanlung**

### CODE REVIEW

File: db/connect.js [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
 
-const connectDB = (url) => {
-  return mongoose.connect(url, {
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true,
-  })
-}
+const connectDB = (url) => mongoose.connect(url);
```
Comment (Indra12345) [inline db/connect.js:3]: You have done this in very simplistic way but often you may have to manage the bugs and waring with options like the instructors did in the video.

File: controllers/jobs.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const createJob = async (req, res) => {
+  req.body.createdBy = req.user.userId;
+  const job = await Job.create(req.body);
+  res.status(StatusCodes.CREATED).json({job});
+};
+
+const updateJob = async (req, res) => res.send("update job");
+
+const deleteJob = async (req, res) => res.send("delete job");
+
```
Comment (Indra12345) [inline controllers/jobs.js:18]: Async is used when you need to use await, while it works if you put async keyword in this function but  
it is not that useful to put 'async' keyword if you do not use 'await' or need to return promise object. In this case,  you're not required.

File: db/connect.js (hunk 1 above)
Comment (Sanlung) [reply db/connect.js:3]: Thank you for noticing and pointing that out. The options in the instructor's boilerplate code for `db/connect.js` would be advisable if I have adopted the instructor's use of Mongoose v. 5.13.2, which came with his boilerplate `package.json`. As I've opted to use Mongoose v. 6.4.6, under which the options are [deprecated](https://mongoosejs.com/docs/migrating_to_6.html) (see "No More Deprecation Warning Options" in bullet list), the options are now by default all set to the specs as spelled out in the instructor's boilerplate, which has become redundant.

File: controllers/jobs.js (hunk 2 above)
Comment (Sanlung) [reply controllers/jobs.js:18]: Thank you for noticing and pointing that out. The `06-jobs-api` segment of the course has two parts spanning week 9 and week 10. The `getAllJobs`, `getJob`, `updateJob` and `deleteJob` controller functionalities had not been implemented in week 9, the progress of which is what you have seen here. The code as you see here is done per the video while still in week 9 and would be fully implemented asynchronous functions in week 10, as could be seen in the next PR.

**Pull request merged**